### PR TITLE
Implement e2e test with an installation of player

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

**Comment by rodrigopavezi** [comment]
[Implement Twitter E2E [5]](https://trello.com/c/QQwSU6LZ/4650-implement-twitter-e2e-5)

**Comment by rodrigopavezi** [comment]
[Slides Widget E2E [5]](https://trello.com/c/eTHOl3lw/5049-slides-widget-e2e-5)

File: test/integration/specs/load-slides-component.js [hunk 1]
```diff
@@ -5,11 +5,21 @@ const commonConfig = require('common-display-module');
...
 const ipc = require("node-ipc");
+const fs = require("fs")
 let localMessagingModule = require('local-messaging-module');
 let testTweets = null;
 
 describe('Slides Component - Integration', () => {
   before(()=>{
+
+    fs.watch = function (filename, options, cb) {
+      console.log("HERE");
```
Comment (jungeunyoon) [inline test/integration/specs/load-slides-component.js]: @rodrigopavezi The only thing is this console log but I imagine it's just here as filler?

Comment (rodrigopavezi) [reply test/integration/specs/load-slides-component.js]: @jungeunyoon nice catch. I was trying to see it was going there. I will remove it. cheers

**Pull request merged**